spec/javascript/components/atoms/StatusPill.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import StatusPill from 'components/atoms/StatusPill'
+
+const getWrapper = (status) => shallow(<StatusPill status={status} />)
+
+describe('StatusPill', () => {
+  test('should render properly with approved status', () => {
```

Comment:
With sufficiently simple components I think snapshot tests do the job fine.

for example this snapshot just looks like this:
```
<div
  className="status-pill is-appealed"
>
  Appealed
</div>
```

That's pretty readable and not much different from a test that specifically tests if the `is-appealed` class is applied